modules/core/shared/src/main/scala/gem/math/EphemerisCoordinates.scala

```diff
@@ -37,6 +37,9 @@ final case class EphemerisCoordinates(
 
 object EphemerisCoordinates {
 
+  val Zero: EphemerisCoordinates =
+    EphemerisCoordinates(Coordinates.Zero, Offset.Zero)
+
```

Comment:
Not sure how the test that uses this value compiled before. I think there was some unnoticed LUBbing happening.